android/app/build.gradle

```diff
@@ -77,7 +79,15 @@ android {
             // Signing with the debug keys for now, so `flutter run --release` works.
             signingConfig signingConfigs.release
         }
+        debug {
+            minifyEnabled false
+        }
```

Comment:
can we disable it?

Reply:
It is needed.
<img width="684" alt="Screenshot 2024-07-23 at 13 57 53" src="https://github.com/user-attachments/assets/34474b9f-2d46-4ea5-8340-fe1ba97f1dd7">

- https://github.com/leancodepl/patrol/issues/1542

Please note that Patrol must be run with debug mode on Android